views/markdown.hbs

```diff
@@ -39,8 +39,8 @@
       <div class="row fs-toolbar mb-3 pad--toolbar d-none">
 
         <div class="upload col-md-2"><label for="picture-input"><div data-toggle="tooltip" data-placement="top" title="add image to your note" class="add--img fa fa-picture-o"></div></label><input onchange="uploadFile()" id="picture-input" type="file" disabled /></div>
-        <div class="upload col-md-2"><label for="audio-input"><div data-toggle="tooltip" data-placement="top" title="add an audio to your note" class="add--audio fa fa-volume-up"></div></label><input id="audio-input" type="file" disabled/></div>
-        <div class="upload col-md-2"><label for="video-input"><div data-toggle="tooltip" data-placement="top" title="add a video to your note" class="add--video fa fa-film"></div></label><input id="video-input" type="file" disabled/></div>
+        <div class="upload col-md-2"><label for="audio-input"><div data-toggle="tooltip" data-placement="top" title="add an audio to your note" class="add--audio fa fa-volume-up"></div></label><input onchange="uploadAudio()" id="audio-input" type="file" disabled/></div>
```

Comment:
@raghuvar-arora-au2  please don't add this onClick method in html use javascript for that.